src/Evolution/Systems/Ccz4/Tags.hpp

```diff
@@ -0,0 +1,69 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/Ccz4/TagsDeclarations.hpp"
+#include "Evolution/Tags.hpp"
+#include "Options/Options.hpp"
+
+class DataVector;
+
+namespace Ccz4 {
+namespace Tags {
+/*!
+ * \brief Auxiliary variable which is analytically the spatial derivative of the
+ * natural log of the lapse
+ *
+ * \details If \f$ \alpha \f$ is the lapse, then we define
+ * \f$ A_i = \partial_i ln(\alpha) = \frac{\partial_i \alpha}{\alpha} \f$.
+ */
+template <size_t Dim, typename Frame>
+struct A : db::SimpleTag {
+  using type = tnsr::i<DataVector, Dim, Frame>;
+};
+
```

Comment:
Note: I did not add a simple tag for the `B^{i}{}_{k}` auxiliary variable in the paper since it appears to simply be the spatial derivative of the shift, and I thought it was possible to just use something like `::Tags::deriv<gr::Tags::Shift<Dim, Frame, DataVector>, tmpl::size_t<Dim>, Frame>`, but let me know if I am mistaken!

Reply:
That sounds right to me 👍 Of course it can be useful to add an alias to abbreviate the long expression, either here or in the place where it's going to be used.

I think you'll need to add more tags eventually though, e.g. for the log conformal factor, the log lapse, and the evolved contracted-christoffel. You can add them later though if you prefer, once you have a better idea which are needed.

Reply:
I think the alias is a good idea! Regarding more tags, I think I will add more simple tags in subsequent PRs, if that's okay with you

Reply:
@nilsleiffischer Do you think that this `DerivShift` simple tag alias belongs in `PointwiseFunctions/GeneralRelativity/Tags.hpp` or `Evolution/Systems/GeneralRelativity/Ccz4/Tags.hpp`?

Reply:
I think in `Ccz4` with a docstring saying it's an auxiliary variable. Then it can easily be made into a struct if necessary.

Reply:
I went ahead and added the alias for the derivative of the shift as well as the log lapse and log conformal factor, since all of those together are necessary for defining the 4 auxiliary variables